pallets/core/src/modules/revoke/impl.rs

```diff
@@ -55,8 +57,9 @@ impl<T: Config> Pallet<T> {
     pub(super) fn remove_registry_(
         RemoveRegistryRaw { registry_id, .. }: RemoveRegistryRaw<T>,
         registry: &mut Option<RevocationRegistry<T>>,
+        _: BTreeSet<PolicyExecutor>,
     ) -> DispatchResult {
-        let registry = registry.take().unwrap();
+        let registry = registry.take().ok_or(Error::<T>::RegistryDoesntExist)?;
```

Comment:
`execute_removable` won't do this check after this PR to allow its usage for inserting new values (when the stored value is `None`).
`execute_view` and `execute` operate over value references rather than `Option<_>` references, so they always have to do this check.